frontend/src/App.js

```diff
@@ -1,24 +1,34 @@
 import logo from './logo.svg';
 import './App.css';
+import Calendar from './components/form/calendar/Calendar';
+import {ExampleForm} from './components/form'
+import { useState } from 'react';
+
+
 
 function App() {
+  const [state,setState] = useState(new Set())
+  var availableTerms= {
+    headers: ["Monday","Tuesday","Wednesday","Thursday","Friday"],
+    rows: [{label:"8:00-9:30",cells:[{ id: 1,isAvailable: true},{id:-1},{id:3},{id:4},{id:5}]},
+            {label:"8:00-9:30",cells:[{ id: 10},{id:2},{id:-1},{id:4},{id:5}]},
+            {label:"8:00-9:30",cells:[{ id: 1},{id:2},{id:3},{id:4},{id:5}]},
+            {label:"8:00-9:30",cells:[{ id: 1},{id:2},{id:3},{id:4},{id:5}]},
+            {label:"8:00-9:30",cells:[{ id: 1},{id:2},{id:3},{id:4},{id:5}]},
+            {label:"8:00-9:30",cells:[{ id: 1},{id:2},{id:3},{id:-1},{id:5}]},
+            {label:"8:00-9:30",cells:[{ id: 1},{id:2},{id:3},{id:4},{id:5}]}]
+}
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <><Calendar availableTerms={availableTerms} selectedTerms ={state} toggleTerm={(id)=>{
+      const newSet = new Set(state.values())
+      if (newSet.has(id)){
+        newSet.delete(id);
+            }
+            else{
+                newSet.add(id);
+             }
+      setState(newSet)
+    }}/></>
   );
```

Comment:
Myśle że nie musimy tego dodawac do App.js. To raczej znajdzie się w kodzie @Antoni535